packages/website/lib/utils.js

```diff
@@ -12,6 +12,31 @@ export const formatTimestamp = timestamp => {
   return formattedDate;
 };
 
+/**
+ * Returns the amount in TiBs if the amount is a multiple of 1024, otherwise returns the amount in GiBs
+ * @param {number} amountInGiB
+ * @returns {string}
+ */
+export const formatAsStorageAmount = amountInGiB =>
+  amountInGiB % 1024 === 0 ? `${amountInGiB / 1024} TiB` : `${amountInGiB} GiB`;
```

Comment:
```suggestion
  amountInGiB > 1024 ? `${amountInGiB / 1024} TiB` : `${amountInGiB} GiB`;
```

Reply:
maybe I'm missing the why here...but this seems like something where once it goes past 1 TiB you only would want to measure it in TiB

e.g. 5.1 TiB vs 5,121 GiB

Reply:
Or are we not worried about this being a problem because it's all from data we can manicure?

Reply:
My thought was it would be a loss of precision, but we don't currently have plans over 1TiB that aren't divisible by 1024.  I'll update it to display an integer in TiB if divisible by 1024, otherwise I'll round to 2 decimal places